include/grpc/impl/codegen/grpc_types.h

```diff
@@ -510,13 +501,9 @@ typedef struct grpc_op {
       struct grpc_byte_buffer **recv_message;
     } recv_message;
     struct {
-      /** ownership of the array is with the caller, but ownership of the
-          elements stays with the call object (ie key, value members are owned
-          by the call object, trailing_metadata->array is owned by the caller).
-          After the operation completes, call grpc_metadata_array_destroy on
-         this
-          value, or reuse it in a future op. */
-      grpc_metadata_array *trailing_metadata;
+      /** Ownership of all metadata and the array stays with the grpc_call */
+      grpc_metadata **trailing_metadata;
+      size_t *trailing_metadata_count;
       grpc_status_code *status;
       grpc_slice *status_details;
```

Comment:
btw, does this slice need to be unref'd  or is it also owned by the call?  Can we pls clarify that in a comment.